### ISSUE TITLE
Document title and language missing from resultant PDF

### ISSUE THREAD

**Issue by smcveigh941** [danfickle/openhtmltopdf#591]
WAVE accessibility evaluation tool and Sortsite report some missing attributes and tags. Many governments require pages to be accessible.

- a missing <title> tag in the head
- a missing lang attribute on the \<html> tag
- missing 'alt' attribute on the \<embed> tag

Can you advise if these can be added to the resultant PDF please? Thanks

See screenshots attached.

<img width="1423" alt="screenshot" src="https://user-images.githubusercontent.com/30229564/97502069-49c7f100-196a-11eb-9589-f6c01aa45f28.png">

<img width="1426" alt="Screenshot 2020-10-28 at 22 04 01" src="https://user-images.githubusercontent.com/30229564/97502084-52202c00-196a-11eb-90d9-1da7556fa0da.png">

<img width="1039" alt="Screenshot 2020-10-28 at 22 11 14" src="https://user-images.githubusercontent.com/30229564/97502171-8562bb00-196a-11eb-93b3-467e1a920284.png">

**Comment by danfickle** [comment]
Hi @smcveigh941,

Accessible PDFs are supported including those attributes. See the [accessibility](https://github.com/danfickle/openhtmltopdf/wiki/PDF-Accessibility-(PDF-UA,-WCAG,-Section-508)-Support) section on the wiki for the details.

**Comment by danfickle** [comment]
Note that in your first two screenshots you are testing the chrome PDF viewer rather than the PDF itself.

**Comment by smcveigh941** [comment]
@danfickle ah! Thanks very much 👍
